docs/VehicleInfo/GetVehicleData/index.md

```diff
@@ -70,6 +70,7 @@ The HMI will have to update this field if the user chooses to reset this value (
 |engineOilLife|Boolean|false||
 |electronicParkBrakeStatus|Boolean|false||
 |cloudAppVehicleID|Boolean|false||
+|handsOffSteering|Boolean|false|To indicate whether driver hands<br>are off the steering wheel|
```

Comment:
`Additional` section is not for placing there params' `Description`
normally, it contains `minvalue`, `maxvalue`, `array`, `minsize`, `maxsize` values.
please, remove the descriptions from `Additional` sections throughout the PR.

Reply:
fixed in 9a59149f05a77b2f17ad729f7a8b3b5f2c9fa831